CleanSwift/MainModule/MainRouter.swift

```diff
@@ -0,0 +1,31 @@
+// MainRouter.swift
+// Copyright © SolovevAA. All rights reserved.
+
+import UIKit
+
+/// Роутер основного экрана
+final class MainRouter: MainRoutingLogic {
+    // MARK: - Public Properties
+
+    weak var viewController: MainViewController?
+
+    // MARK: - Private Properties
+
+    private var dataStore: MainDataStore
+
+    // MARK: - Initializers
+
+    init(dataStore: MainDataStore) {
+        self.dataStore = dataStore
+    }
+
+    // MARK: - Public Methods
+
+    func routeToDetail() {
+        let titleImage = dataStore.titleImage
```

Comment:
```suggestion
        let titleImage = dataStore.titleImage
```
заругают за создание константы

Reply:
Не, тут не константа это свойство из датаСтор